From cecba1668bc8607112a6a2b802c82b6313a0d2b4 Mon Sep 17 00:00:00 2001
From: Adrian Alonso <aalonso@freescale.com>
Date: Tue, 19 Jun 2012 15:37:56 -0500
Subject: [PATCH] gstplaybin2: rawvideo support

* Enable support for rawvideo
* Support direct rendering for mfw_v4lsink plugin

Signed-off-by: Adrian Alonso <aalonso@freescale.com>
---
 gst/playback/gstplaybin2.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst/playback/gstplaybin2.c b/gst/playback/gstplaybin2.c
index e01ff6c..9c638ae 100644
--- a/gst/playback/gstplaybin2.c
+++ b/gst/playback/gstplaybin2.c
@@ -456,7 +456,7 @@ struct _GstPlayBinClass
 #define DEFAULT_SUBURI            NULL
 #define DEFAULT_SOURCE            NULL
 #define DEFAULT_FLAGS             GST_PLAY_FLAG_AUDIO | GST_PLAY_FLAG_VIDEO | GST_PLAY_FLAG_TEXT | \
-                                  GST_PLAY_FLAG_SOFT_VOLUME
+                                  GST_PLAY_FLAG_SOFT_VOLUME | GST_PLAY_FLAG_NATIVE_VIDEO
 #define DEFAULT_N_VIDEO           0
 #define DEFAULT_CURRENT_VIDEO     -1
 #define DEFAULT_N_AUDIO           0
--
1.7.10.2
